pyccl/cls.py

```diff
@@ -145,12 +130,12 @@ def get_internal_function(self, cosmo, function, a):
             function (:obj:`str`): Specifies which function to evaluate. Must
                 be one of
                     'dndz': number density
-                    'bz', 'bias': bias
-                    'sz', 'm_bias': magnification bias
-                    'rfz', 'red_fraction': red fraction
-                    'baz', 'a_bias': intrinsic alignment bias
-                    'wL', 'window_lensing': weak lensing window function
-                    'wM', 'window_magnif': magnification window function
+                    'bias': bias
+                    'mag_bias': magnification bias
+                    'red_frac': red fraction
+                    'ia_bias': intrinsic alignment bias
+                    'lensing_win': weak lensing window function
+                    'mag_win': magnification window function
```

Comment:
I like a lot of the changes to the names here (in addition to the class name changes).  But I suspect this breaks quite a few of the jupyter notebooks that are in the repo `examples` directory.  You should probably fix them to conform to the new API as part of this PR.

Reply:
I made a second pr for this. I fixed everything there. I was going to do it in two passes.

Reply:
Happy to put those changes back here. It is just a lot.

Reply:
Ahhh Just saw this. Great!

Reply:
I tagged you on the PR with the changes to the examples. I reran all of them too.